### DISCUSSION TITLE
I need some help with the data analysis of my howler monkey research!

### DISCUSSION THREAD

**Discussion by Jort-Mans** [community/community#129123] [Programming Help]
### Body

Hello!
I am currently situated in Paraguay, where I did research on howler monkeys. I am now in the data analysis fase of my project and pretty stuck.
Background of the project:
I played 5 different sounds (Ship horn, elephant, howling of conspecifics, people talking and a silent control) all once to 9 different monkey groups. The monkey groups were all divided into 3 different habitat types (natural, peri-urban and urban). Within the groups subjects were sampled based on age-sex category (e.g. AM-adult male, JF-juvenile female) and a couple of different reactions were measured, like vocalization, vigilance or movement towards the speaker.

Data collected:
My data was entered in excel with every trial as a row, with columns for the sound, habitat, visibility of the different age-sex categories when doing the trial and a column for every behavior type performed by every age-sex category like this:

![behavdata](https://github.com/community/community/assets/173318460/cb6fbbf1-d698-46e8-8acf-749ef6a50d40)

I added a column were I divided the behavior performed by a age-sex category, through the amount that age-sex category was visible, creating for example the 'CorTowardsAM' column (cor for corrected).

I then transformed this data to look like this:

![lijstdata](https://github.com/community/community/assets/173318460/fd6274ad-dc21-4926-bcd1-605db9421124)

where value is the corrected amount of times a behavior in a particular situation was performed by a particular age-sex category.
Now I did not really know how to analyse this. The main thing I want to know is if there is a difference in reaction between the habitat types and to a lesser extent between the sound types. I felt like I had to correct however for the differences in groups and age-sex categories. Therefore after some looking I came across making a linear mixed effects model, with habitat, behavior and sound as fixed effects and group and subject as random effects. I did mine like this in Rstudio:

![modeloutput](https://github.com/community/community/assets/173318460/519d68ad-f84f-4a1f-8d12-046aad82d27a)

I have a hard time figuring out if this is a correct way to analyse my data. I also have a hard time figuring out how to intrepret these results.
I made this plot from the data too:

![barplotsoundhabitat](https://github.com/community/community/assets/173318460/910da5b3-c4aa-4ba4-8f33-f0c6934e71ff)

I was hoping if people with more knowledge about how data works could help me a little. Am I on the right track? Is the kind of data I have generally used in a linear mixed effects model? If not, how can I analyse it? If a linear mixed effects model is right, what do I do with the output and do I need to check assumptions beforehand?

Thanks a lot for everyone wanting to look at this way too long post!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Jort-Mans, thanks for being a part of the GitHub Community and sharing this with us!

Boosting for visibility as you posted this last week. While we may not always have users who are able to help with your issue, I hope you continue to post and engage in Discussions 😄
